test/JDBC/input/test_windows_login-vu-prepare.mix

```diff
@@ -165,3 +165,52 @@ GO
 -- test for windows login with password --> should throw error
 CREATE LOGIN [babel\adbabel] from windows with password='1234';
 GO
+
+CREATE LOGIN [ba,bel\username] from windows;
```

Comment:
Can we add one testcase for '.' as well ? since we are allowing it right now

Reply:
Test case from Jira is missing.